internal/files/files.go

```diff
@@ -1,45 +1,32 @@
 package files
 
 import (
-	"fmt"
 	"sort"
-	"strings"
 
-	"github.com/goreleaser/nfpm"
 	"github.com/goreleaser/nfpm/glob"
 )
 
-// FileToCopy describes the source and destination of one file to copy into a
-// package and whether it is a config file.
+// FileToCopy describes the source and destination
+// of one file to copy into a package.
 type FileToCopy struct {
 	Source      string
 	Destination string
-	Config      bool
 }
 
-// ToCopy lists all of the real files to be copied into the package.
-func ToCopy(info *nfpm.Info) ([]FileToCopy, error) {
+// Expand gathers all of the real files to be copied into the package.
+func Expand(filesSrcDstMap map[string]string) ([]FileToCopy, error) {
```

Comment:
I think Expand is a fitting name since this function basically just expands the globs in a source->dest map.

Reply:
agreed!